### ISSUE TITLE
Should color contrast matcher exclude `aria-disabled=true` on parent elements?

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#1766]
The color-contrast-matcher does not matches elements if they have any parent node with `aria-disabled=true`. We should research if `aria-disabled` propagates to children properly and if the child is really disabled.

If not, we should check the element still and then mark it as needs review instead of failing if an ancestor has `aria-disabled=true`.
